docs/contributing/doc_guide.rst

```diff
@@ -1417,3 +1417,71 @@ version of the package that can be revisited later.
 .. _Sphinx's templating page: https://www.sphinx-doc.org/en/master/templating.html
 .. _style overrides: https://docs.readthedocs.io/en/stable/guides/adding-custom-css.html
 .. _warns: https://numpydoc.readthedocs.io/en/latest/format.html#warns
+
+.. ----------------------
+.. Nested inline literals
+.. ----------------------
+
+.. A workaround for nested inline literals so that the filename will get
+   formatted like a file but will be a link. In the text, these get used
+   with the syntax for a substitution followed by an underscore to
+   indicate that it's for a link: |docs/_static|_
+
+.. For these workarounds, if the replacement is something in single back
+   ticks (e.g., `xarray`), then it should also be added to
+   nitpick_ignore_regex in docs/conf.py so that it doesn't get counted
+   as an error in a nitpicky doc build (e.g., tox -e doc_build_nitpicky).
```

Comment:
This was copied verbatim from `docs/common_links.rst` so it's already been reviewed in a previous pull request.